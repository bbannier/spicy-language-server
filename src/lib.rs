#![feature(str_strip)]

#[macro_use]
extern crate pest;

pub mod lsp;
pub mod options;
pub mod spicy;
