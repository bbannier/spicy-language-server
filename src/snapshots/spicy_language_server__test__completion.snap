---
source: src/lib.rs
expression: "server.completion(CompletionParams\n{\n    text_document_position:\n    TextDocumentPositionParams::new(TextDocumentIdentifier::new(uri.clone()),\n    Position::new(u32::MAX, 0),), work_done_progress_params:\n    WorkDoneProgressParams::default(), partial_result_params:\n    PartialResultParams::default(), context: None,\n}).await"
---
Ok(
    None,
)
